lib/oli_web/live/progress/student_resource_view.ex

```diff
@@ -106,7 +106,7 @@ defmodule OliWeb.Progress.StudentResourceView do
         nil
 
       ra ->
-        Oli.Repo.preload(ra, :resource_attempts)
+        Oli.Repo.preload(ra, resource_attempts: [activity_attempts: [:part_attempts]])
```

Comment:
If we create that new `preload_activity_part_attempts` core function, we could use it here as well
instead of further polluting this live view with calls directly to `Oli.Repo`

Reply:
I have left this as it was before. I have removed the preloads